### PR TITLE
To Do list: interactive list

### PR DESCRIPTION
**Pull request by Camilovelag**
# Good day! 🌤️

This pull request is intended to create a "Todo List" project structure using webpack.
It is powered by @Camilovelag.

## 🛠️ Changes made

✔️ Add a new JavaScript file to update the task `completed `status.
✔️ Add event listener to the checkbox (change).
✔️ Update items object's value for completed key upon user actions.
✔️ Implement a function for the "Clear all completed" button.
✔️ Store the updated array of items in local storage.

We're expecting feedback.

Regards!  👏👏👏

done

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -4,29 +4,26 @@
...
-
-<!-- task item template (keep in to ease layout refactor/modifications)-->
-
-      <!-- <li class="task-item b-bottom box">
-        <input type="checkbox" class="checkbox" checked>
-        <span class="task-text">Task 1 Lorem, ipsum dolor sit amet consectetur adipisicing elit.</span>
-        <button class="delete">X</button>
-      </li> -->
-
+      <!-- taskList appended here -->
```
Comment (rustam-tolipov) [inline dist/index.html:24]: - [ ] Kindly remove unused commented code from your project.

File: dist/index.html [hunk 2]
```diff
@@ -4,29 +4,26 @@
...
     </ul>
-    <div class="box clr-all">Clear all completed</div>
+    <button type="button" class="box clr-all">Clear all completed</button>
```
Comment (rustam-tolipov) [inline dist/index.html:26]: [OPTIONAL] I would suggest adding MIT license to your project.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @Camilovelag, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Camilovelag** [comment]
## Hi @Rustamxon7 

I really appreciate your time and effort in your feedback.

I've checked the required changes, but it seems you reviewed the `dist/index.html `from the `main` branch. Below you can find the code section from the `interactive` branch. As you can see, non-used code isn't present.

Probably it could be a misunderstanding in the 'files changed' tab interpretation, because in the permalink you sent me is noticeable these red highlighted sections were deleted in `interactive` compared to the `main`. 😲

I'll be open to new feedback. 😄😄😄

done

https://github.com/Camilovelag/Todo_list/blob/632be77b392c5f61c798e78dfb51c73d5a0bfb10/dist/index.html#L17-L29

**Comment by rustam-tolipov** [comment]
Ohh, my bad bro! Let me fix it for you!

**Review by rustam-tolipov — APPROVED** [review]
## Hello @Camilovelag👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**